plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -1580,7 +1580,9 @@ static bool amf_hevc_init(void *data, obs_data_t *settings)
 	int rc = get_hevc_rate_control(rc_str);
 
 	set_hevc_property(enc, RATE_CONTROL_METHOD, rc);
-	set_hevc_property(enc, ENABLE_VBAQ, true);
+	if (rc != 0) {
```

Comment:
Should this not check for `AMF_VIDEO_ENCODER_HEVC_RATE_CONTROL_METHOD_CONSTANT_QP` directly? (Damn that's a long key)

Reply:
`AMF_VIDEO_ENCODER_HEVC_RATE_CONTROL_METHOD_CONSTANT_QP` will just be replaced with the literal value `0`by the compiler, there is no performance difference. But it's more "correct" and should the enum ever change (god forbid) it will continue to work.

Reply:
Alright, understood :)

Will try to figure out how to use that approach.